developer-docs/README.md

```diff
@@ -36,7 +36,7 @@ Updates are also on its way to include more self-service tools, along with docum
 ## Project getting started guide
 Kubernetes resources (including your applications!) are always deployed into a particular namespace.
 These namespaces provide separation between the different projects hosted on the platform.
-We also recommend that each project has one namespace per environment. For example if the project name were *foo* the namespaces might be *foo-dev*, *foo-uat*, *foo-preprod*. For instructions on creating new namespaces and other relevant resources on getting started can be found below on [ACP How-to Docs](#ACP-How-to-Docs)
+We also recommend that each project has one namespace per service. For example if you had a project named `foo`, and two services that were named `hello` and `world`, the namespaces might be `foo-hello`, `foo-world`. For instructions on getting new namespaces and other relevant resources on getting started can be found below: [ACP How-to Docs](#ACP-How-to-Docs)
```

Comment:
Based on discussions yesterday, I think we said a project service could have 0 or more namespaces (e.g. for different environments)?

Also, it would be good to refer to "service" as "project service", from now on. Also, the first time someone reads that they probably won't know what that means exactly – we should probably think about having a doc explaining projects structures in the hub, or an FAQ entry in the hub that we can link to from here? (note: you don't have to address this in this PR – we can resolve this later).